### ISSUE TITLE
Next stage: Auto add email to member list?

### ISSUE THREAD

**Issue by Rumyra** [WebVJs/webvjs.com#3]
At the moment one of the slack admins has to invite members, a new user can't just sign up. This isn't necessarily a bad thing, but it means someone has to contact us directly.

Would a form be a good solution? Request an invite, that emails admins with the persons deets and we add them to slack...

**Comment by davemackintosh** [comment]
I think a slack button like the one on Zach's website would be better, people can just sign up that way http://zachinglis.herokuapp.com/south-england-slack/

**Comment by Rumyra** [comment]
Yeh this seems like a really good idea... Zach uses https://github.com/rauchg/slackin

For this we need to:
• Register a domain
• Set up a heroku app to host just the slack.ourdomain.com for the 'request an invite page' (I still think the main site can hold up on gh-pages)
• Set up as per the slackin instructions

**Comment by davemackintosh** [comment]
Yeah, it works really well for the south central slack still see a handful of people join every few days. Pretty sure it doesn't cost Zach anything either so the only costing would be the domain.
